.github/workflows/docs.yml

```diff
@@ -0,0 +1,74 @@
+name: "Documentation Preview"
+
+on:
+  push:
+    paths-ignore:
+      - '**'
+      - '!docs/**'
+      - '!.github/workflows/docs.yml'
+    branches-ignore:
+      - main
+      - master
+  pull_request:
+    paths-ignore:
+      - '**'
+      - '!docs/**'
+      - '!.github/workflows/docs.yml'
+    branches:
+      - '**'
+
+jobs:
+  deploy-preview:
+    name: Build & Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Website Source
+        uses: actions/checkout@v3
+        with:
+          repository: python-poetry/website
+          ref: website-allow-local-version-builds
```

Comment:
```suggestion
```

Once the website PR is merged this suggestion can be commited, squashed and merged.